lokeshsingh3721/ClassVault
# fix: fixes the avatar logo in navbar

##What does this PR do ?
This fixes the avatar logo in the navbar .

Fixes #1 
![Screenshot (520)](https://github.com/lokeshsingh3721/ClassVault/assets/145474757/fff527a0-0047-4219-95ce-0dc58a3dad55)

##Type of changes
-[] Bug Fix

## How should this be tested
-[] Go to Homepage 
-[] then signup 
-[] then show avatar logo in right side corner fix

##Mandatory task
-[X] Make sure you have you self-reviewed the code .
